<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StokHareket</class>
 <widget class="QWidget" name="StokHareket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>647</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferDefault</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget_alt">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>541</width>
     <height>421</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>30</number>
   </property>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Fatura Numara</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Fatura Tarih</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Satıcı</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Alıcı</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Adet</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_HareketAra">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>ARA</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_ust">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>641</width>
     <height>61</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>1</number>
   </property>
   <property name="columnCount">
    <number>6</number>
   </property>
   <row/>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Stok Adı</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Marka</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Model</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Toplam Miktar</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Açıklama</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_sil">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>600</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SİL</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_hareketara">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
